core/modules/views/plugins/views_plugin_style_grid.inc

```diff
@@ -38,7 +41,36 @@ class views_plugin_style_grid extends views_plugin_style {
       '#required' => TRUE,
       '#min' => 1,
       '#step' => 1,
-      '#weight' => -2,
+    );
+    $form['cell_min_width'] = array(
+      '#type' => 'number',
+      '#title' => t('Minimum grid cell width'),
+      '#field_suffix' => 'px',
+      '#attributes' => array('style' => 'width: 6em;'),
+      '#description' => t('The minimum width of the grid cells. If the grid container becomes narrow, the grid cells will reflow onto the next row as needed.'),
+      '#default_value' => $this->options['cell_min_width'],
+      '#required' => TRUE,
+      '#min' => 1,
+      '#states' => array(
+        'invisible' => array(
+          ':input[name="style_options[deprecated_unresponsive]"]' => array('checked' => TRUE),
+        ),
+      ),
```

Comment:
Should this also be hidden if using the other deprecated (table-based) grid?

Reply:
ok